packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -114,14 +120,14 @@ function Wayfinding({ onStartDirections, onBack, directionsToLocation, direction
      */
     function locationClickHandler(location) {
         if (activeSearchField === searchFieldIdentifiers.TO) {
-            decorateLocation(location).then(location => {
-                setDestinationLocation(location);
-                toFieldRef.current.setDisplayText(location.properties.name);
+            decorateLocation(location).then(selectedLocation => {
```

Comment:
Why the change from `location` to `selectedLocation`?

Reply:
It was a bit confusing because we had `location` everywhere, and it made it more difficult to distinguish between them

Reply:
Could we call it `decoratedLocation` instead? I'm not really following the `selected` wording 🤔

Reply:
✅